airqo-platform/AirQo-frontend
# [NetManager] Clean codebase

#### Summary of Changes (What does this PR do?)
- Removes unused directories/files from netmanager

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Testing this PR is mainly to ensure no page is broken (especially css)
* Fetch this branch locally
* Install requirements `npm install`
* Start the `npm run stage-mac` (MacOs) or `npm run stage-pc` (Windows platform)

#### What are the relevant tickets?
- [PLAT-470](https://airqoteam.atlassian.net/browse/PLAT-470)

#### Screenshots (optional)
